values.yaml

```diff
@@ -45,16 +53,14 @@ minio:
   ingress:
     enabled: true
     hostname: data.outline.yourdomain.tld
-    certManager: false
```

Comment:
This isn't needed anymore in the new minio version?

Reply:
The default value is false, so it's not needed. And I doubt that someone who is installing this chart want to use the cert-manager of minio.